packages/perspective-viewer/src/less/computed_expression_widget.less

```diff
@@ -11,7 +11,6 @@
 
 :host {
     display: none;
-    //font-family: var(--interface--font-family, @sans-serif-fonts);
```

Comment:
We should make comments that don't have s space trailing the comment operator a lint error so we clean these up!